### ISSUE TITLE
Trap tab-focus when opening the modal

### ISSUE THREAD

**Issue by noahmanger** [presidential-innovation-fellows/vote-2016#126]
Small thing I noticed: for accessibility and usability purposes, when opening the modal, `focus` should be sit inside the modal and you shouldn't be able to tab outside the modal. 

This is a nice little library we're using on beta.fec.gov if it helps: https://github.com/edenspiekermann/a11y-dialog

Keep being awesome! 🇺🇸
